.gitignore

```diff
@@ -25,6 +25,9 @@ devsetup/ccloud/
 /demo/kube-ip.conf
 /demo/.demofiles
 
+/application-npr.yml
```

Comment:
Too specific files - why should anyone have such files locally? Could you replace this with a more generic pattern (`application-*.yml`), and exclude the ones being needed (e.g. `!/application-demo.yml`)?